back-end/api/quickstart/migrations/0020_auto_20210215_2314.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.5 on 2021-02-15 23:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('quickstart', '0019_auto_20210215_2214'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='author',
+            name='_id',
+            field=models.CharField(max_length=128, primary_key=True, serialize=False),
```

Comment:
Is the PR missing a migration? It seems that the newest changes to the models don't include your change to the author model.

Reply:
what change to the author model? I don't think I changed anything. The changes I made are in `signals.py` but didn't cause any migrations